proto/brpb.proto

```diff
@@ -898,6 +898,48 @@ message MetaEdit {
     bool destruct_self = 4;
 }
 
+// RewrittenTableID records a sort of modification over the SSTs during restoring.
+// 
+// When "backing up" this "restored" SST, we want to backup what it was "restored".
+// But in some cases, SSTs are not restored "as is", TiKV may rewrite or truncate its
+// content by the client's request. 
+message RewrittenTableID {
+    // The table ID in the backup data.
+    int64 upstream_of_upstream = 1;
+    // The rewritten table ID during restoring.
+    int64 upstream = 2;
+}
+
+// ExtraFullBackup is created by BR restoring to a cluster with PITR enabled.
+// This is uploaded by the BR CLI.
+message ExtraFullBackup {
```

Comment:
What do you think of the name `LogBackupInjestedSST` or other similar options. 
I feel like in our code base there are a lot vague terms and we need to remember what they mean when reading the code, making a bit hard to understand. I personally like to name as explicit as possible. Let me know your thoughts. 
Feel free to ignore the comment if you think otherwise!